mrrc/config.py

```diff
@@ -65,13 +62,27 @@ def get_aws_bucket(self) -> str:
         return bucket
 
     def __val_or_default(self, section: dict, key: str, default=None):
+        if not section:
+            return default
```

Comment:
@yma88 This is designed to do, because the order of fetching bucket is: sysenv var -> configuration file "mrrc->bucket" -> default. You can check the command.__get_bucket() method for this order.
For the section "mrrc", if it is not defined in conf file, I think we should return the default bucket, just like the code does.